src/Workspaces/Core/Portable/Recommendations/AbstractRecommendationServiceRunner.cs

```diff
@@ -108,8 +107,7 @@ private ImmutableArray<ITypeSymbol> GetTypeSymbols(ImmutableArray<ISymbol> candi
                 {
                     if (method.Parameters.Length > ordinalInInvocation)
                     {
-                        var methodParameterSymbool = method.Parameters[ordinalInInvocation];
-                        var type = methodParameterSymbool.Type;
+                        var type = method.Parameters[ordinalInInvocation].Type;
                         // If type is <see cref="Expression{TDelegate}"/>, ignore <see cref="Expression"/> and use TDelegate.
                         if (type is INamedTypeSymbol expressionSymbolNamedTypeCandidate &&
                             Equals(expressionSymbolNamedTypeCandidate.OriginalDefinition, expressionSymbol))
```

Comment:
Why not the .Equals instance form? If that's binding to object.Equals it's kinda strange to read.